native/rust/README.md

```diff
@@ -0,0 +1,64 @@
+# Rust components in Alluxio
+
+This module is home to Alluxio native components implemented in Rust.
+
+## Project layout
+
+Each Maven submodule, except for the `archetype` module, contains a Rust crate. It may also include necessary Java
+wrappers for the crate.
+The Rust source files are located in `src/main/rust`, and the Java wrappers are located in `src/main/java`.
+After a build is complete, the build artifacts, both Java class files and Rust native binaries, are stored within
+the `target/classes` directory.
+
+## Building the modules
+
+To build Rust sources, you will need the Rust toolchains. The recommended way is to use
```

Comment:
it would be good to update the `alluxio-maven` image that is used as the build environment to include the Rust toolchains. the dockerfile for the image is located in `dev/github/Dockerfile-jdk8`. there are also dockerfiles for jdk11 and jdk17; it would be nice to update those too but they are currently not in use.

Reply:
I added rust toolchain to the dockerfile:

https://github.com/Alluxio/alluxio/blob/9205036b7773f84a962eab6b8c8ee6fa48483dca/dev/github/Dockerfile-jdk8#L40-L41

but it does not seem to work:

```
2023-09-25T17:54:35.0392570Z 17:54:35.038 [INFO] 17:54:35.037 [ERROR] -> [Help 1]
2023-09-25T17:54:35.0398031Z 17:54:35.039 [INFO] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.questdb:rust-maven-plugin:1.1.1:build (release-build) on project alluxio-rust-example: Rust's `cargo` not found in PATH=/usr/local/openjdk-8/bin:/usr/local/go/bin:/usr/local/openjdk-8/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin.
2023-09-25T17:54:35.0608292Z 17:54:35.060 [INFO] 
2023-09-25T17:54:35.0620795Z 17:54:35.061 [INFO] See https://www.rust-lang.org/tools/install
```

looks like the rust toolchain is not properly installed. Does it require re-building the image so that github CI can pick up the change?

Reply:
yes requires rebuilding the image and subsequently bumping up the image version in `dev/github/run_docker.sh`